tensorflow/python/ops/math_ops_test.py

```diff
@@ -39,8 +39,8 @@ class ReduceTest(test_util.TensorFlowTestCase):
 
   def testReduceAllDims(self):
     x = np.array([[1, 2, 3], [4, 5, 6]], dtype=np.int32)
+    y_tf = self.evaluate(math_ops.reduce_sum(x))
```

Comment:
The changes in this file actually change the test logic. Unlike the previous file, the context manager is placing the reduce_sum operation on GPU. I would suggest reverting the changes in this file.

If not, you might consider putting the numpy assertions outside of the context manager, although I don't think that's a useful change:

```python
with test_util.device(use_gpu=True):
  y_tf = self.evaluate(math_ops.reduce_sum(x))	
self.assertEqual(y_tf, 21)	
```